forge-gui/res/cardsfolder/upcoming/expel_the_unworthy.txt

```diff
@@ -0,0 +1,11 @@
+Name:Expel the Unworthy
+ManaCost:1 W
+Types:Sorcery
+K:Kicker:2 W
+A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target creature with mana value 3 or less | ValidTgts$ Creature.cmcLE3 | TargetMin$ X | TargetMax$ X | SubAbility$ DBChangeZone | SpellDescription$ Choose target creature with mana value 3 or less. If this spell was kicked, instead choose target creature. Exile the chosen creature, then its controller gains life equal to its mana value.
+SVar:DBChangeZone:DB$ ChangeZone | TargetMin$ Y | TargetMax$ Y | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Creature | Condition$ Kicked | SubAbility$ DBGainLife
+SVar:DBGainLife:DB$ GainLife | Defined$ TargetedController | LifeAmount$ Z 
```

Comment:
StackDescription$ SpellDescription | SpellDescription$ then its controller gains life equal to its mana value.